src/main.rs

```diff
@@ -210,37 +209,54 @@ fn to_ca_certs(ca_certs_pem: &Vec<Vec<u8>>) -> Result<rustls::RootCertStore> {
     Ok(root_cert)
 }
 
-fn init_tracing(path: &Path) -> Result<WorkerGuard> {
-    if !path.exists() {
-        bail!("Path not found {path:?}");
-    }
-
+/// Initializes the tracing subscriber.
+///
+/// If `some_dir` is `None` or the runtime is in debug mode, logs will be printed to stdout.
```

Comment:
I think it would be easier to understand if we just used `dir`. Using `some_dir` might mislead people into thinking it represents an array of paths rather than a single path.

Reply:
Besides, since `Some` has a special meaning in Rust, it would be safer to use it with caution. Plus, such a name looks quite unusual to me.

Reply:
That makes sense but the `dir` seems too generic.
How about using `log_dir`?

Reply:
> That makes sense but the `dir` seems too generic. How about using `log_dir`?

It works as well.

Reply:
The name has been changed to `log_dir`. Thank you.